packages/app-portal/src/systems/Chains/fuel/services/txFuelToEth.ts

```diff
@@ -260,6 +264,26 @@ export class TxFuelToEthService {
     };
   }
 
+  static calculateDelayBasedOnTransactionTimeToFinalize(
```

Comment:
this can be a bit more generic?
```suggestion
  static calculateDelayBasedOnTimeRemaining(
```